nc: Improve title and some examples (#2790)
tldr-pages/tldr: pages/common/nc.md

# nc

> > Netcat is a versatile utility for working with tcp or udp data.

Reviewer: ```suggestion
> Netcat is a versatile utility for working with TCP or UDP data.
```

I'd capitalize TCP and UDP here while we're at it.

Author: Thanks, updated.

`nc -w {{timeout_in_seconds}} {{ipaddress}} {{port}}`

> 

Reviewer: I think this is good as it is. The command `nc -l port < file` is not actually sending a file automatically. It is serving it waiting for someone to connect to the specified port.

Reviewer 2: Yeah, let's not change this.

Author: Reverted

`nc -q {{timeout}} {{ip_address}}`

> - Scan open ports on host:

Reviewer 3: Perhaps this will read better?

```suggestion
- Scan the open ports of a specified host:
```

Author: Thanks, updated.

`nc -v -z {{ip_address}} {{port}}`

> - Proxy and port forwarding, makes a remote connection on local port and prints response:

Reviewer: This is incorrect, nothing is printed from `nc -l local_port | nc hostname remote_port`.

I would reword it like this:

```suggestion
- Proxy and port forwarding, forward data from a local TCP port to the given remote host:
```

Author: Updated.

> - Proxy and port forwarding, forward data from a local TCP port to the given remote host:

Reviewer 2: Maybe we can remove "port forwarding" too just as we did in the earlier example. So perhaps - "Act as proxy and forward data from ..."

Reviewer: I agree, looks cleaner.

Author: Updated.
